packages/core/src/messages/responses/leaderboard/leaderboard-length.ts

```diff
@@ -0,0 +1,66 @@
+import {SdkError} from '../../../errors';
+import {ResponseBase, ResponseError, ResponseSuccess} from '../response-base';
+
+/**
+ * Parent response type for a leaderboard length request.  The
+ * response object is resolved to a type-safe object of one of
+ * the following subtypes:
+ *
+ * - {Success}
+ * - {Error}
+ *
+ * `instanceof` type guards can be used to operate on the appropriate subtype.
+ * @example
+ * For example:
+ * ```
+ * if (response instanceof LeaderboardLength.Error) {
+ *   // Handle error as appropriate.  The compiler will smart-cast `response` to type
+ *   // `LeaderboardLength.Error` in this block, so you will have access to the properties
+ *   // of the Error class; e.g. `response.errorCode()`.
+ * }
+ * ```
+ */
+export abstract class Response extends ResponseBase {}
+
+class _Success extends Response {
+  private readonly _length: number;
+  constructor(length: number) {
+    super();
+    this._length = length;
+  }
+
+  /**
+   * Returns the length of the leaderboard
+   * @returns {number}
+   */
+  public length(): number {
```

Comment:
is there a chance that this needs to be `bigint` too? might be a question for the backend team

Reply:
i see you are returning `bigint` for `rank` so it definitely seems like this one would need to be `bigint`?

Reply:
Good catch, the proto returns length as `bigint` as well

https://github.com/momentohq/client-protos/blob/main/proto/leaderboard.proto#L147